testing/k3s/APKBUILD

```diff
@@ -0,0 +1,67 @@
+# Contributor: Oleg Titov <oleg.titov@gmail.com>
+# Maintainer: Oleg Titov <oleg.titov@gmail.com>
+pkgname=k3s
+pkgver=0.4.0
+pkgrel=0
+pkgdesc="Lightweigt Kubernetes. 5 less than k8s."
+url="https://k3s.io"
+arch="all"
+license="Apache-2.0"
+depends="containerd"
+makedepends="go linux-headers"
+install=""
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/rancher/k3s/archive/v$pkgver.tar.gz"
+builddir="$srcdir/src/github.com/rancher/$pkgname"
+
+prepare() {
+	export GOPATH="$srcdir"
+
+	mkdir -p $(dirname $builddir)
+	mv "$srcdir"/$pkgname-$pkgver "$builddir"/
+
+	default_prepare
+}
+
+build() {
+	msg2 "Prepare GOPATH and GOBIN..."
+	export GOPATH="$srcdir"
+	export GOBIN="$GOPATH/bin"
+
+	msg2 "Building k3s..."
+	go build -o k3s
+
+	msg2 "Building kubectl..."
+	go build -o kubectl ./cmd/kubectl
+
+	msg2 "Building hypercube..."
+	go build -o hyperkube ./vendor/k8s.io/kubernetes/cmd/hyperkube
+}
+
+check() {
```

Comment:
The [policy][] officially changed that checks should use test suites from upstream. So just running the command in some way no longer suffices. So if no test suite is available from upstream, the package should be marked as such.

[policy]:https://wiki.alpinelinux.org/wiki/APKBUILD_Reference#options

Reply:
I have removed checks, thank you for bringing to my attention new corresponding policy to follow.